### PR TITLE
Build improvements

### PR DESCRIPTION
**Pull request by luizirber**
- Add `py35` to Travis and Tox configuration
- Run `codecov` for coverage reports
- Add pep8 check to builds, fix pep8 errors
- Update PR checklist and add GitHub templates

## Checklist

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make clean diff-cover` If it introduces new functionality, is it tested?
- [x] `make format diff_pylint_report doc` Is it well formatted?
- [x] Is it documented in the `ChangeLog`?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by luizirber** [comment]
OK to test, Jenkins

**Comment by luizirber** [comment]
Ready for review and merge @ctb @standage @betatim

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+See [our development docs](https://screed.readthedocs.io/en/latest/dev/).
+
+Be sure to copy and paste the [checklist](https://screed.readthedocs.io/en/latest/dev/coding-guidelines-and-review.html#checklist) in the Pull-Request comment
```
Comment (betatim) [inline .github/CONTRIBUTING.md]: With the template setup the checklist should just appear in the comment. Don't know if you want to adjust the text to say that.

**Comment by betatim** [comment]
LGTM. Feel free to merge depending on what you want to do about that sentence in the contributing.md

**Pull request merged**